dcos_installer/upgrade.py

```diff
@@ -50,12 +50,20 @@
 fi
 
 SKIP_CHECKS=false
+VERBOSE=false
 
 if [[ $# -ne 0 ]]; then
-  if [[ "$1" = "--skip-checks" ]]; then
-     echo "Skipping checks"
-     SKIP_CHECKS=true
-  fi
+    for var in "$@";
+    do
+        if [[ "$var" = "--skip-checks" ]]; then
+          echo "Skipping checks"
+          SKIP_CHECKS=true
+        fi
+        if [[ "$var" = "--verbose" ]]; then
+          echo "Verbose mode on"
+          VERBOSE=true
+        fi
+    done
```

Comment:
The indentation in this stanza is inconsistent, it's a mix of 4- and 2-space indents. Should be 4-space indents to be consistent with the rest of the script.